vitrina/templates/confirm_delete.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block current_title %}{{ current_title }}{% endblock %}
+
+{% block content %}
+    <div class="columns">
+        <div class="column is-half">
+            <form method="post">
+                {% csrf_token %}
+                <strong>
+                    <p >{% translate "Ar tikrai norite ištrinti" %} "{{ object }}"?</p>
```

Comment:
![image](https://user-images.githubusercontent.com/297583/192768535-f4ad7cb3-39df-4224-b327-ee7db30de4df.png)

A more descriptive name should be show, instead of „Representative object (3)“. Consider adding `Representative.__str__`.